bundle/metadata/annotations.yaml

```diff
@@ -4,9 +4,9 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: netobserv-operator
-  operators.operatorframework.io.bundle.channels.v1: v1.0.x
-  operators.operatorframework.io.bundle.channel.default.v1: v1.0.x
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.25.2
+  operators.operatorframework.io.bundle.channels.v1: v0.2.x
+  operators.operatorframework.io.bundle.channel.default.v1: v0.2.x
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.22.2
```

Comment:
Shall we use latest operator-sdk by default or stick to a given version?

Reply:
I think we don't need to enforce a specific version, as long as the produced bundle stays the same, minus that particular line ?
In the new github action to verify the bundle, this line is ignored (as well as the "createdAt" line in CSV)

Reply:
given that, if someone uses a too old operator-sdk that produces different bundle, the github action will throw a failure, so that will force us to upgrade operator-sdk when necessary